spec/pushmi_pullyu/aip/creator_spec.rb

```diff
@@ -2,25 +2,15 @@
 
 RSpec.describe PushmiPullyu::AIP::Creator do
   let(:workdir) { 'tmp/creator_spec' }
-  let(:options) do
-    { workdir: workdir,
-      fedora: { url: 'http://www.example.com:8983/fedora/rest',
-                base_path: '/dev',
-                user: 'fedoraAdmin',
-                password: 'fedoraAdmin' },
-      solr: { url: 'http://www.example.com:8983/solr/development' } }
-  end
   let(:noid) { '9p2909328' }
-  let(:aip_file) { "#{workdir}/#{noid}.tar" }
-  let(:mock_download_data) { "spec/fixtures/aip_download/#{noid}" }
+  let(:aip_file) { "#{aip_folder}.tar" }
+  let(:aip_folder) { "#{workdir}/#{noid}" }
```

Comment:
Add a aip_folder variable. Here we can use these variables instead of expecting the hardcoded "tmp/creator_spec/9p2909328.tar" or "tmp/creator_spec/9p2909328" multiple times down below. If we ever decide to change theses, its now a one line code change instead of multiple lines.